triage/2024-07-30.md

```diff
@@ -0,0 +1,289 @@
+
+- [#128360 Rollup of 6 pull requests](https://github.com/rust-lang/rust/pull/128360)
+- [#128301 Rollup of 3 pull requests](https://github.com/rust-lang/rust/pull/128301)
+- [#128265  Perform instsimplify before inline to eliminate some trivial calls ](https://github.com/rust-lang/rust/pull/128265)
+- [#128253 Rollup of 9 pull requests](https://github.com/rust-lang/rust/pull/128253)
+- [#128169 Rollup of 5 pull requests](https://github.com/rust-lang/rust/pull/128169)
+- [#128155 Rollup of 8 pull requests](https://github.com/rust-lang/rust/pull/128155)
+- [#128104 Not lint pub structs without pub constructors intentionally](https://github.com/rust-lang/rust/pull/128104)
+- [#127998 Rollup of 9 pull requests](https://github.com/rust-lang/rust/pull/127998)
+- [#127865 Rollup of 8 pull requests](https://github.com/rust-lang/rust/pull/127865)
+- [#127528 Replace ASCII control chars with Unicode Control Pictures](https://github.com/rust-lang/rust/pull/127528)
+- [#127486 Rollup of 5 pull requests](https://github.com/rust-lang/rust/pull/127486)
+- [#127442 Try to fix ICE from re-interning an AllocId with different allocation contents](https://github.com/rust-lang/rust/pull/127442)
+- [#127439 Uplift elaboration into `rustc_type_ir`](https://github.com/rust-lang/rust/pull/127439)
+- [#127172 Make `can_eq` process obligations (almost) everywhere](https://github.com/rust-lang/rust/pull/127172)
+- [#127096 Rollup of 11 pull requests](https://github.com/rust-lang/rust/pull/127096)
+- [#127076 Rollup of 6 pull requests](https://github.com/rust-lang/rust/pull/127076)
+- [#126852 Also get `add nuw` from `uN::checked_add`](https://github.com/rust-lang/rust/pull/126852)
+- [#126591 [perf] More span update benchmarking](https://github.com/rust-lang/rust/pull/126591)
+- [#126578 Account for things that optimize out in inlining costs](https://github.com/rust-lang/rust/pull/126578)
+- [#126134 Rollup of 11 pull requests](https://github.com/rust-lang/rust/pull/126134)
+- [#126024 Do not use global caches if opaque types can be defined](https://github.com/rust-lang/rust/pull/126024)
+- [#125968 Store the types of `ty::Expr` arguments in the `ty::Expr`](https://github.com/rust-lang/rust/pull/125968)
+- [#125824 Rollup of 7 pull requests](https://github.com/rust-lang/rust/pull/125824)
+- [#125144 Rollup of 6 pull requests](https://github.com/rust-lang/rust/pull/125144)
+- [#125120 Rollup of 7 pull requests](https://github.com/rust-lang/rust/pull/125120)
+- [#125069 Extend SCC construction to enable extra functionality ](https://github.com/rust-lang/rust/pull/125069)
+- [#124905 Allow optimizing `u32::from::<char>`.](https://github.com/rust-lang/rust/pull/124905)
+- [#124700 Remove an unnecessary cast](https://github.com/rust-lang/rust/pull/124700)
+- [#124417 Make early lints translatable](https://github.com/rust-lang/rust/pull/124417)
+- [#124241 Rollup of 5 pull requests](https://github.com/rust-lang/rust/pull/124241)
+
+#### Nags requiring follow up
+
+TODO: Nags
```

Comment:
maybe we can remove this as well?

Reply:
yeah I did so in follow-up commit c12b75fb04eb51cb5f0a6eaae2398e2425aafbc7